grpc_message_getter.js

```diff
@@ -0,0 +1,292 @@
+    const contractCertHolderIdEncoded = new TextEncoder('utf-8').encode(
+        this.certHolderId);
+    const contractCertVersionEncoded = new TextEncoder('utf-8').encode(
+        this.certVersion);
+    const contractIdEncoded = new TextEncoder('utf-8').encode(
+        contractId);
+    const buffer = new Uint8Array(
+        contractCertHolderIdEncoded.byteLength
+        + contractCertVersionEncoded.byteLength
+        + contractIdEncoded.byteLength);
+
+    let offset = 0;
+    buffer.set(contractCertHolderIdEncoded, offset);
+    offset += contractCertHolderIdEncoded.byteLength;
+    buffer.set(contractCertVersionEncoded, offset);
+    offset += contractCertVersionEncoded.byteLength;
+    buffer.set(contractIdEncoded, offset);
+    offset += contractIdEncoded.byteLength;
+
+    const signatureEncoded = new TextEncoder('utf-8').encode(
+        await this.signer.sign(buffer));
+
+    const resizedBuffer = new Uint8Array(offset + signatureEncoded.byteLength);
+    resizedBuffer.set(buffer);
+    resizedBuffer.set(signatureEncoded, offset);
+
+    return resizedBuffer
+  }
+
+  /**
+   * @param {number} contractId
+   * @param {Object} contractArgument
+   * @param {Object} [functionArgument=undefined]
+   * @return {Uint8Array}
+   */
+  async getContractExecutionRequest(contractId, contractArgument,
+      functionArgument) {
+    const contractIdEncoded = new TextEncoder('utf-8').encode(
+        contractId.toString());
+    const contractArgumentEncoded = new TextEncoder('utf-8').encode(
+        JSON.stringify(contractArgument));
+    const contractCertHolderIdEncoded = new TextEncoder('utf-8').encode(
+        this.certHolderId);
+    const contractCertVersionEncoded = new TextEncoder('utf-8').encode(
+        this.certVersion);
+    const functionArgumentEncoded = new TextEncoder('utf-8').encode(
+        JSON.stringify(functionArgument));
+    const buffer = new Uint8Array(
+        contractIdEncoded.byteLength
+        + contractArgumentEncoded.byteLength
+        + contractCertHolderIdEncoded.byteLength
+        + contractCertVersionEncoded.byteLength
+        + functionArgumentEncoded.byteLength);
+
+    let offset = 0;
+
+    buffer.set(contractIdEncoded, offset);
+    offset += contractIdEncoded.byteLength;
+    buffer.set(contractArgumentEncoded, offset);
+    offset += contractArgumentEncoded.byteLength;
+    buffer.set(contractCertHolderIdEncoded, offset);
+    offset += contractCertHolderIdEncoded.byteLength;
+    buffer.set(contractCertVersionEncoded, offset);
+    offset += contractCertVersionEncoded.byteLength;
+    buffer.set(functionArgumentEncoded, offset);
+    offset += functionArgumentEncoded.byteLength;
+
+    const signatureEncoded = new TextEncoder('utf-8').encode(
+        await this.signer.sign(buffer));
+
+    const resizedBuffer = new Uint8Array(offset + signatureEncoded.byteLength);
+    resizedBuffer.set(buffer);
+    resizedBuffer.set(signatureEncoded, offset);
+
+    return resizedBuffer
```

Comment:
Here, the buffer is resized (increased) so that we can fit all the `signatureEncoded` bytes into  it.